src/components/pages/Navbar/Navbar.js

```diff
@@ -14,16 +13,30 @@ import ApplyButton from './NavbarFeatures/ApplyButton';
 import LogoutButton from './NavbarFeatures/LogoutButton';
 import MentorPopover from './NavbarFeatures/MentorPopover';
 import { useAuth0 } from '@auth0/auth0-react';
+import { getCurrentUser } from '../../../state/actions/userProfile/getCurrentUser';
+import { acceptMentee } from '../../../state/actions/mentor/acceptMenteeStatus';
+import { mentorInfo } from '../../../state/actions/mentor/postMentorInfo';
+import { useDispatch } from 'react-redux';
 
 const { Header } = Layout;
 
-const Navbar = ({ isAuthenticated, userProfile, getProfile }) => {
+const Navbar = ({
+  isAuthenticated,
+  userProfile,
+  getProfile,
+  currentUser,
+  State,
+}) => {
   const [profilePic] = useState('https://joeschmoe.io/api/v1/random');
   const [user, setUser] = useState({});
   const [modal, setModal] = useState(false);
   const [toggleStatus, setToggleStatus] = useState(false);
   const { logout } = useAuth0();
 
+  const dispatch = useDispatch();
+
+  console.log(State);
```

Comment:
I would remove this console.log from the file!

Reply:
Thank you austin, and thank you very much for your attentive eye to this, I just fixed it and removed the unused code from the code base on my last commit now.